clean/clean/clean_mock.yaml

```diff
@@ -0,0 +1,29 @@
+cleaners:
```

Comment:
I'm going to approve and merge. but `clean/clean` folder is a bit weird. maybe better to have better name?. please fix later. @GerriorL